packages/atomic/src/components/common/interface/bindings.ts

```diff
@@ -21,9 +42,15 @@ export interface CommonBindings<
   /**
    * Global state for Atomic
    */
-  store: StoreCreation;
+  store: Store;
   /**
    * A reference to the atomic interface element.
    */
   interfaceElement: InterfaceElement;
 }
+
+export type AnyBindings = CommonBindings<
```

Comment:
typescript still blows my mind

Reply:
Yeah, it's powerful.
The type system is actually a complete programming language in itself :D 

https://dev.to/gypsydave5/fizz-buzz-in-typescript-7ip